packages/web/src/components/list/utils.js

```diff
@@ -34,7 +34,7 @@ const getAggsQuery = (query, props) => {
 };
 
 const getCompositeAggsQuery = (query, props, after) => {
-	const clonedQuery = { ...query };
+	const clonedQuery = { ...query, ...(props.defaultQuery && props.defaultQuery()) };
```

Comment:
Shouldn't we do this merging of queries at the end of these functions (right before we return the final query)? Then, it will ensure that if `aggs` is present in the defaultQuery, it will over-write the generated `aggs` query.

Reply:
Also, we should only take `query` and `aggs` from `defaultQuery` here, since we apply other query-options on the root query [here](https://github.com/appbaseio/reactivesearch/blob/cf59fe76590d55ad432b541dc10bfed6f5b0c4fc/packages/web/src/components/list/MultiList.js#L383).

What do you think?

Reply:
cool, will update the PR